script/build.js

```diff
@@ -424,6 +414,13 @@ smith.use(redirect({
   '/education/apply-for-education-benefits/': '/education/apply/'
 }));
 
+// TODO: what is the earliest place in this script where we can guarantee all HTML files have been
+// generated? metalsmith-linkcheck only checks HTML files
+// https://github.com/gchallen/code.metalsmith-linkcheck
+smith.use(linkcheck({
+  failMissing: true
```

Comment:
Argh--just looked at the plugin code, and it turns out this is the only way to configure it to fail, but it *only* checks internal links and fails if they're broken. If external links are broken, it writes them to file and passes. 😠  I'm ready to give up on this library, but will file some issues against it and see if the developer has any interest in adding some config options